pkg/appconsts/global_consts.go

```diff
@@ -29,8 +29,8 @@ const (
 
 	// DefaultUpgradeHeightDelay is the number of blocks after a quorum has been
 	// reached that the chain should upgrade to the new version. Assuming a block
-	// interval of 12 seconds, this is 7 days.
-	DefaultUpgradeHeightDelay = int64(7 * 24 * 60 * 60 / 12) // 7 days * 24 hours * 60 minutes * 60 seconds / 12 seconds per block = 50,400 blocks.
+	// interval of 6 seconds, this is 7 days.
+	DefaultUpgradeHeightDelay = int64(7 * 24 * 60 * 60 / 6) // 7 days * 24 hours * 60 minutes * 60 seconds / 6 seconds per block = 100,800 blocks.
 )
```

Comment:
Hmm I think this is doable because users of the earlier version will still crash in any case when the state machine transitions to v3 (even if nodes in the network have different values of the `DefaultUpgradeHeightDelay`

Reply:
Isn't doable? Or is?

Thanks for the review!